.github/workflows/lua.yml

```diff
@@ -40,23 +40,33 @@ jobs:
   luacheck:
     name: "Builtin Luacheck and Unit Tests"
     runs-on: ubuntu-20.04
+
     steps:
-    - uses: actions/checkout@v3
-    - uses: leafo/gh-actions-lua@v9
-      with:
-        luaVersion: "5.1.5"
-    - uses: leafo/gh-actions-luarocks@v4
-
-    - name: Install luarocks tools
-      run: |
-        luarocks install --local luacheck
-        luarocks install --local busted
-
-    - name: Run checks (builtin)
-      run: |
-        $HOME/.luarocks/bin/luacheck builtin
-        $HOME/.luarocks/bin/busted builtin
-
-    - name: Run checks (devtest)
-      run: |
-        $HOME/.luarocks/bin/luacheck --config=games/devtest/.luacheckrc games/devtest
+
+      - uses: actions/checkout@v3
+      - uses: leafo/gh-actions-lua@v9
+        with:
+          luaVersion: "5.1.5"
+      - uses: leafo/gh-actions-luarocks@v4
+
+      - name: Install LuaJIT
+        run: |
+          cd $HOME
+          git clone https://github.com/LuaJIT/LuaJIT/
```

Comment:
I'd rather skip that one test on LuaJIT than this

also why does this entire file show as changed? whitespace?

Reply:
It would be to minimize external dependencies in CI but I supposed it's not that important.